.travis.yml

```diff
@@ -43,6 +43,9 @@ script:
   - tar xf cmake-3.13.4-Linux-x86_64.tar.gz
   - export PATH=${PWD}/cmake-3.13.4-Linux-x86_64/bin:${PATH}
 
+  # Use the correct virtualenv
```

Comment:
Why does installing virtualenv suddenly cause it to use the correct virtualenv? A bit more of a comment here explaining the fix would be great.

Reply:
Updated comment.